TLSharp.Core/TelegramClient.cs

```diff
@@ -25,14 +25,24 @@ namespace TLSharp.Core
     public class TelegramClient : IDisposable
     {
         private MtProtoSender sender;
+        private AuthKey _key;
         private TcpTransport transport;
-        private string apiHash = String.Empty;
+        private string apiHash = "";
         private int apiId = 0;
         private Session session;
         private List<TLDcOption> dcOptions;
         private TcpClientConnectionHandler handler;
         private DataCenterIPVersion dcIpVersion;
 
+        private bool _looping = true;
```

Comment:
please rename variable to not have underscore (to be consistent with the rest of the vars)